src/assets/i18n/en.json

```diff
@@ -336,7 +336,7 @@
   "T_TIME_LEFT": "Time will leave",
   "T_INVITES": "Invites",
   "T_INVITE_EMAIL": "Email",
-  "T_INVITE_GROUPS": "Add users to groups",
+  "T_INVITE_GROUPS": "Groups",
```

Comment:
It seems like this translation should be removed and the translation in a template should be replaced with actual `T_GROUPS` key.
Same for all languages.

Reply:
:heavy_check_mark: